deploy/crds/capabilities.3scale.net_v1beta1_openapi_cr_url.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: capabilities.3scale.net/v1beta1
+kind: OpenAPI
+metadata:
+  name: example-openapi
```

Comment:
Can we change the name of this sample or the configmap one?
The reason I'm asking this is due to in the new operator-sdk version you cannot have two samples with the same name